### DISCUSSION TITLE
Restrict access for the editors to some records only [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2333] [Discord Threads]
> 🙋‍♂️ **esteve**  
## Hello ! Is it possible to restrict access for the editors to some records only ? E.g. the apple salesman shall only be able to view & edit the records with the “apple” attribute and the oranges salesman shall only be able to view & edit the records with the “orange” attributes ? Thanks a lot  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/982277955722051614/IMG_0257.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi, i donì't think that is possible atm  
#  
> 🙋‍♂️ **FrancyMak**  
#### <@&682859083799199814> do you confirm?  
#  
> 👨‍💻 **pranavxc**  
#### not possible at the moment  
#  
> 🙋‍♂️ **FrancyMak**  
#### Let me try a workaround  
#  
> 🙋‍♂️ **FrancyMak**  
#### It is not possible even with the workaround.  
#  
> 🙋‍♂️ **esteve**  
#### Can I restrict access to some tables only then ? Is that just a “view” with edit capabilities? That might be a workaround  
#  
> 💁 **o1lab**  
#### you can restrict access to tables & views - but only at UI level  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/meta-management#ui-access-control  
#  
> 🙋‍♂️ **esteve**  
#### Thank you, I’m going to try that  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/982277956548321280)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
